lib/scholar/stats.ex

```diff
@@ -0,0 +1,196 @@
+defmodule Scholar.Stats do
+  @moduledoc """
+  Statistical functions
+
+  All of the functions in this module are implemented as
+  numerical functions and can be JIT or AOT compiled with
+  any supported `Nx` compiler.
+  """
+
+  import Nx.Defn
+
+  general = [
+    axis: [
+      type: {:or, [:non_neg_integer, :atom]},
+      default: 0,
+      doc: """
+      Axis to calculate the operation. If set to `nil` then
+      the operation is performed on the whole tensor.
+      """
+    ],
+    keep_axis: [
+      type: :boolean,
+      default: false,
+      doc: "If set to true, the axis which is reduced is left."
+    ]
+  ]
+
+  moment_schema =
+    general ++
+      [
+        moment: [
+          type: :pos_integer,
+          default: 1,
+          doc: "Order of central moment that is returned."
+        ]
+      ]
+
+  skew_schema =
+    general ++
+      [
+        bias: [
+          type: :boolean,
+          default: true,
+          doc: "If false, then the calculations are corrected for statistical bias."
+        ]
+      ]
+
+  kurtosis_schema =
+    general ++
+      [
+        bias: [
+          type: :boolean,
+          default: true,
+          doc: "If false, then the calculations are corrected for statistical bias."
+        ],
+        variant: [
+          type: {:in, [:fisher, :pearson]},
+          default: :fisher,
+          doc:
+            "If :fisher then Fisher's definition is used, if :pearson then Pearson's definition is used."
+        ]
+      ]
+
+  @moment_schema NimbleOptions.new!(moment_schema)
+  @skew_schema NimbleOptions.new!(skew_schema)
+  @kurtosis_schema NimbleOptions.new!(kurtosis_schema)
+
+  @doc """
+  Calculate the nth moment about the mean for a sample.
```

Comment:
Use third person singular voice and we typically describe the return value at the top, instead of a separate section. Please replicate the changes in other functions! :)

```suggestion
  Calculates the nth moment about the mean for a sample.
  
  It returns the appropriate moment along the given axis or whole tensor.
```